O/OpenFHE/README.md

```diff
@@ -0,0 +1,8 @@
+The OpenFHE library works with both 64-bit and 128-bit integers.
+
+This package builds the default 64-bit version.
+
+To build the 128-bit version, use OpenFHE_128.
+
+Most of the build steps for OpenFHE and OpenFHE_128 are the same, so the shared parts are in the
+common.jl script.
```

Comment:
Can you maybe add links to the `OpenFHE_128` directory and the `common.jl` script for easy access?